packages/cita-sdk/package.json

```diff
@@ -36,7 +36,8 @@
   },
   "dependencies": {
     "@cryptape/cita-signer": "^2.4.1",
-    "web3": "1.0.0-beta.37"
+    "web3": "1.0.0-beta.37",
+    "fstream": "^1.0.12"
```

Comment:
Please lock the version without the version range.

Reply:
Ok, I have modified it. https://github.com/cryptape/cita-sdk-js/pull/201/commits/43bf116728a6774d8721c4ea259b666e3026817c